src/transformers/models/bigscience176b/modeling_bigscience176b.py

```diff
@@ -569,12 +691,19 @@ def __init__(self, config):
         super().__init__(config)
 
         self.embed_dim = config.hidden_size
+        dtype = getattr(torch, config.dtype)
+        assert dtype in [torch.bfloat16, torch.float32, torch.float, torch.float16]
+
+        self.word_embeddings = nn.Embedding(config.vocab_size, self.embed_dim, dtype=dtype)
+        # self.word_embeddings = nn.Embedding(config.vocab_size, self.embed_dim)
+        # self.word_embeddings_layernorm = MixedFusedLayerNorm(self.embed_dim, dtype=dtype, eps=config.layer_norm_epsilon)
```

Comment:
clean up a bit this init func

Reply:
Done [here](https://github.com/younesbelkada/transformers/commit/93d270a906a54f3f584d6cbe84196c7088069fa0)!